wiki/markdown/curl-examples.md

```diff
@@ -93,52 +163,76 @@ This way of entering the `backendurl` may be the easiest way. It works with GET,
 
 Note: It is not possible to add the `backendurl` parameter as a JSON parameter.
 
-## Endpoints bridged from front-end to back-End
+
+## <a id="download" />/download
+
 <table>
     <tr>
         <th>Context Path</th>
         <th>Type</th>
         <th>Explanation</th>
     </tr>
     <tr>
-        <td>/auth</td>
+        <td>/download/subscriptionsTemplate</td>
         <td>GET</td>
-        <td>Check if LDAP security is enabled</td>
+        <td>Description</td>
     </tr>
     <tr>
-        <td>/auth/checkStatus</td>
+        <td>/download/rulesTemplate</td>
         <td>GET</td>
-        <td>Used to check the back-end status.</td>
+        <td>Description</td>
     </tr>
     <tr>
-        <td>/subscriptions</td>
+        <td>/download/eventsTemplate</td>
         <td>GET</td>
-        <td>Fetches all available subscriptions from the back-end</td>
-    </tr>
-    <tr>
-        <td>/subscriptions</td>
-        <td>POST</td>
-        <td>Request to add a new subscription included as a JSON object</td>
+        <td>Description</td>
     </tr>
+</table>
+
+The EI front-end supports these endpoints. More information can be found in the [EI back-end documentation](https://github.com/eiffel-community/eiffel-intelligence/blob/master/wiki/markdown/Download-Files.md)
+
+#### Some curl examples
+
+    curl -X GET -H "Content-type: application/json" http://localhost:8080/download/rulesTemplate
```

Comment:
I think you need to specify a download file to make it clear, and those 3 commands looks almost the same so maybe have 1 and add a description.

Reply:
Updated the last curl example with a filename :)